packages/turbo-repository/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@turbo/repository",
-  "version": "0.0.1-canary.1",
+  "version": "0.0.1-canary.2",
```

Comment:
should this be happening in a PR like this or through the release workflow. ok for a greenfield thing, but generally the release cut should be separate

Reply:
I'm slowly ramping up the automation for this. The previous two releases were manual version bumps, this one was at least automated file changes. Next step is to wire it to GH actions.